src/lib/server/endpoints/google/endpointVertex.ts

```diff
@@ -43,7 +62,7 @@ export function endpointVertex(input: z.input<typeof endpointVertexParametersSch
 		const parameters = { ...model.parameters, ...generateSettings };
 
 		const generativeModel = vertex_ai.getGenerativeModel({
-			model: model.id ?? model.name,
+			model: model.name ?? model.id,
```

Comment:
revert to model.id ?? model.name, until the nested model object is actually used.